hexa/ui/datagrid.py

```diff
@@ -309,29 +307,11 @@ def context(self, model: DjangoModel, grid: Datagrid):
         return data
 
 
```

Comment:
What if I want to use `BooleanColumn` in another grid? In this case, you shouldn't delete it just because you don't use it ;). Sometimes, it's the way to go but not in this case.